dds/DCPS/RTPS/GuidGenerator.cpp

```diff
@@ -186,7 +199,13 @@ GuidGenerator::interfaceName(const char* iface)
   }
 
   ACE_OS::close(h);
-  return found ? 0 : -1;
+
+  if (found) {
+      interface_name_ = iface;
+      return 0;
```

Comment:
```suggestion
    interface_name_ = iface;
    return 0;
```

Reply:
This one appears correct.  Why remove the space?  The 'if' is indented once, so the lines after the bracket should be indented twice.